streaming/multimodal/convert/webvid/inside_to_outside.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Convert an MP4-inside MDS dataset to an MP4-outside MDS dataset."""
+
+import os
+from argparse import ArgumentParser, Namespace
+
+from tqdm import tqdm
+
+from streaming import MDSWriter, StreamingDataset
+
+out_columns = {
```

Comment:
nit: `columns` instead of `out_columns` ?

Reply:
i figure we use out_columns because this script has input columns which are different